care_scribe/tasks/scribe.py

```
@@ -87,200 +129,173 @@ def chat_message(provider=plugin_settings.SCRIBE_API_PROVIDER, role="user", text
-        # recalculate used quota. This prevents edge cases where quota was exceeded last month and this is the first request this month
-        if facility_quota:
-            facility_quota.calculate_used()
-        if user_quota:
-            user_quota.calculate_used()
+    try:
+        base_prompt = form.prompt or BASE_PROMPT
+        base_prompt = base_prompt.replace("{current_date_time}", current_time.isoformat())
 
-        error = None
+        if not form.audio_file_ids and not form.document_file_ids:
+            raise ScribeError("No audio or documents associated with the Scribe. Your upload might have failed.")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)